### DISCUSSION TITLE
【新功能建议】增加 TTS 语音朗读功能

### DISCUSSION THREAD

**Discussion by shension** [immersive-translate/immersive-translate#381] [Ideas]
非常棒的工具！感谢开发者们的付出。

在此提一个锦上添花的功能建议：增加英文 TTS 语音朗读功能，TTS 引擎最好能允许用户自定义。

我设想的使用方式有两种：
1. 段落朗读：鼠标悬停到英文段落时能出现朗读图标，点击朗读图标后可调用 TTS 引擎朗读。
2. 整篇文章朗读：在文章顶部或其他位置提供一个整篇朗读的图标，点击后朗读整篇英文内容。

关于 TTS 引擎，我简单评测过 [Google Cloud](https://cloud.google.com/text-to-speech) 和微软的引擎，感觉Google的会更胜一筹，特别是其中有一个 Voice name 为 en-US-Neural2-C 的声音非常好听。

**Comment by theowenyoung** [comment]
嗯～ 不错，可以从段落朗读做起，在每个译文部分可以提供一些快捷操作，比如朗读原文

**Comment by solider245** [comment]
最好再加上原文朗读与收集功能就很完美了，背诵了原文之后再去理解英文，记忆速度会更快
